src/components/home/Landing.jsx

```diff
@@ -8,8 +8,10 @@ const Landing = () => {
     <div className="w-full relative mb-[3%]">
       <Image src={image1} alt="Background image 1" className="w-screen" />
       <div className="flex justify-center flex-col w-1/2 h-full bg-swim-blue-200/70 top-0 absolute pl-[5%]">
-        <p className="font-bold text-white md:text-6xl mb-4">UCR Club Swim</p>
-        <p className="font-semibold text-white md:text-2xl">
+        <p className="font-bold text-white text-2xl md:text-6xl mb-4 mt-[10%] md:mt-[0%]">
+          UCR Club Swim
+        </p>
+        <p className="font-semibold text-white text-xs md:text-2xl">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod
         </p>
```

Comment:
![image](https://github.com/acm-ucr/ucr-swim-website/assets/91030482/caf8df39-1576-4f5a-9d03-ca00d7814288)

button overflows in some screens
example : iPhone 11 Pro iOS 14.6

Reply:
you can probably remove the top/bottom padding/margining and use gap instead.
https://tailwindcss.com/docs/gap